bootstrap-salt.sh

```diff
@@ -3589,6 +3589,9 @@ install_fedora_stable_post() {
 
 install_fedora_git_deps() {
     if [ -n "$_PY_EXE" ] && [ "$_PY_MAJOR_VERSION" -eq 3 ]; then
+        if [ "$DISTRO_MAJOR_VERSION" -eq 31 ]; then
+            __install_tornado_pip "${_PY_EXE}"|| return 1
```

Comment:
This function seems to be not universal and does not support Fedora properly.

The script returns this: ` Error: Unknown repo: 'epel'`, because of the call to `yum` with `epel` enabled instead of `dnf`. I guess we need more precise logic or, even better, make `__yum_install_noinput()` function more generic to support `dnf` (without epel) as well.

Reply:
Have you looked at this @Ch3LL ?

Reply:
i haven't had time to come back to this quit yet, i know @bryceml did some work here though

I can probably come back to this later this week/next week